src/state_machine.zig

```diff
@@ -1988,6 +2056,15 @@ fn check(test_table: []const u8) !void {
                 const reply_actual_buffer = try allocator.alignedAlloc(u8, 16, 4096);
                 defer allocator.free(reply_actual_buffer);
 
+                context.busy = true;
+                context.state_machine.prefetch(
+                    TestContext.callback,
+                    op,
+                    commit_operation,
+                    request.items,
+                );
+                while (context.busy) context.storage.tick();
+
```

Comment:
That's neat! I struggled to write a unit test for expired transfers because I avoided touching the storage 👍